pkgs/tools/typesetting/tex/texlive/bin.nix

```diff
@@ -24,16 +24,16 @@ let
       # FIXME: a proper mirror, though tarballs.nixos.org saves this case ATM
       # http://146.185.144.154/texlive-2016
       # + "/texlive-${year}0523b-source.tar.xz";
-        "http://ftp.math.utah.edu/pub/tex/historic/systems/texlive/${year}/texlive-${year}0524-source.tar.xz";
-      sha256 = "1amjrxyasplv4alfwcxwnw4nrx7dz2ydmddkq16k6hg90i9njq81";
+        "http://ftp.math.utah.edu/pub/tex/historic/systems/texlive/${year}/texlive-${year}0414-source.tar.xz";
+      sha256 = "0khyi6h015r2zfqgg0a44a2j7vmr1cy42knw7jbss237yvakc07y";
     };
 
     patches = [
-      (fetchurl {
-        name = "texlive-poppler-0.59.patch";
-        url = https://git.archlinux.org/svntogit/packages.git/plain/trunk/texlive-poppler-0.59.patch?h=packages/texlive-bin&id=6308ec39bce2a4d735f6ff8a4e94473748d7b450;
-        sha256 = "1c4ikq4kxw48bi3i33bzpabrjvbk01fwjr2lz20gkc9kv8l0bg3n";
-      })
+       (fetchurl {
+         name = "texlive-poppler-0.64.patch";
+         url = https://git.archlinux.org/svntogit/packages.git/plain/trunk/texlive-poppler-0.64.patch?h=packages/texlive-bin;
+         sha256 = "0443d074zl3c5raba8jyhavish706arjcd80ibb84zwnwck4ai0w";
+       })
```

Comment:
Why did you add the extra whitespace?

Reply:
That wasn't intentional, maybe my editor's autoindent. Will clean it up before this is merged.